pkg/api/customization/catalog/validator.go

```diff
@@ -38,3 +43,10 @@ func validateURL(pathURL string) error {
 	}
 	return nil
 }
+
+func validateHelm3Version(helmVersion string) bool {
```

Comment:
I think it would make more sense to return true if the version is valid or to name the function something like invalidHelm3Version

Reply:
Sounds good - will do!